deploy/install.sh

```diff
@@ -115,55 +112,69 @@ getLatestRelease() {
 downloadFile() {
     LATEST_RELEASE_TAG=$1
 
-    OS_ARCH="${OS}-${ARCH}"
-    RADIUS_CLI_ARTIFACT="rad"
-    DOWNLOAD_BASE="https://get.radapp.dev/tools/rad"
-    DOWNLOAD_URL="${DOWNLOAD_BASE}/${LATEST_RELEASE_TAG}/${OS_ARCH}/${RADIUS_CLI_ARTIFACT}"
+    RADIUS_CLI_ARTIFACT="${RADIUS_CLI_FILENAME}_${OS}_${ARCH}"
+    DOWNLOAD_BASE="https://github.com/${GITHUB_ORG}/${GITHUB_REPO}/releases/download"
+    DOWNLOAD_URL="${DOWNLOAD_BASE}/${LATEST_RELEASE_TAG}/${RADIUS_CLI_ARTIFACT}"
 
     # Create the temp directory
-    RADIUS_TMP_ROOT=$(mktemp -dt Radius-install-XXXXXX)
+    RADIUS_TMP_ROOT=$(mktemp -dt radius-install-XXXXXX)
     ARTIFACT_TMP_FILE="$RADIUS_TMP_ROOT/$RADIUS_CLI_ARTIFACT"
 
-    if [ "$RADIUS_HTTP_REQUEST_CLI" == "curl" ]; then
-        if ! curl --output /dev/null --silent --head --fail "$DOWNLOAD_URL"; then
-            echo "ERROR: The specified release version: $LATEST_RELEASE_TAG does not exist."
-            exit 1
-        fi
-    else
-        if ! wget --spider "$DOWNLOAD_URL" 2>/dev/null; then
-            echo "ERROR: The specified release version: $LATEST_RELEASE_TAG does not exist."
-            exit 1
-        fi
-    fi
-
-
-    echo "Downloading ${DOWNLOAD_URL}..."
+    echo "Downloading $DOWNLOAD_URL ..."
     if [ "$RADIUS_HTTP_REQUEST_CLI" == "curl" ]; then
         curl -SsL "$DOWNLOAD_URL" -o "$ARTIFACT_TMP_FILE"
     else
         wget -q -O "$ARTIFACT_TMP_FILE" "$DOWNLOAD_URL"
     fi
 
     if [ ! -f "$ARTIFACT_TMP_FILE" ]; then
-        echo "failed to download ${DOWNLOAD_URL}..."
+        echo "failed to download $DOWNLOAD_URL ..."
```

Comment:
```suggestion
        echo "failed to download ${DOWNLOAD_URL}..."
```

There was an extra period between the URL and the periods

Reply:
Updated